packages/shared/lib/sdk/sync.ts

```diff
@@ -620,8 +641,17 @@ export class NangoAction {
         return this.nango.triggerAction(providerConfigKey, connectionId, actionName, input) as T;
```

Comment:
What about triggerAction? If not possible to dryRun can we just log something instead?

Reply:
This is something I've mentioned to Bastien quite a few times and is a larger product question and outside the scope of this issue IMO.

Reply:
this is an issue because you can wait for the result?

Reply:
Yeah, it is an issue mainly because the whole premise of an action is to interact with an external API.

Reply:
make sense